src/components/Sections/IconsMeaning/IconsMeaning.scss

```diff
@@ -0,0 +1,77 @@
+@import "../../../themes/mixins";
+@import "../../../themes/variables";
+
+.container {
```

Comment:
Does it look good on different screens when you choose "responsive" under "inspect"? Pixel values for positioning and dimensions, such as width, height, and left properties may cause issues. These fixed values may not adapt well to different screen sizes, leading to layout inconsistencies. To ensure better responsiveness, consider using relative units like percentages or viewport units (vw, vh) for sizing and positioning elements.